tidy3d/components/geometry/primitives.py

```diff
@@ -215,6 +222,105 @@ def _only_middle_for_infinite_length_slanted_cylinder(cls, val, values):
             )
         return val
 
+    def to_polyslab(self, num_pts_radius: int = N_PTS_CYLINDER_POLYSLAB, **kwargs) -> PolySlab:
+        """Convert instance of ``Cylinder`` into a discretized version using ``PolySlab``.
+
+        Parameters
+        ----------
+        num_pts_radius : int = 100
+            Number of points in the radius of the discretized polyslab.
```

Comment:
The hard-coded default is 51 now.

Also, do you mean number of points in the *circumference*?

Reply:
fixed this, and yea circumference. will change all mentions of radius to circumference..